.github/workflows/Dockerfile

```diff
@@ -0,0 +1,24 @@
+# GitHub action dockerfile
+# Requires docker experimental features as buildx and BuildKit so not suitable for developers regular use.
+# https://docs.docker.com/develop/develop-images/build_enhancements/#to-enable-buildkit-builds
+
+###########################
+# Build binaries stage
+###########################
+FROM --platform=$BUILDPLATFORM golang:1.16.15-alpine3.15 AS build
```

Comment:
I think go 1.16 is no longer supported.  Really should use 17 (or 18).

Reply:
Done.